applications/serial_lte_modem/src/lwm2m_carrier/Kconfig

```diff
@@ -12,7 +12,8 @@ if SLM_CARRIER
 
 config SLM_CARRIER_APP_DATA_BUFFER_LEN
 	int "Size of the buffer for setting application data"
-	default 512
+	default 768
+	range 0 768
```

Comment:
Just a wild idea: Could the value be made to depend on `CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE`?

Reply:
It would probably make more sense, however in this case, the maximum range is dictated by a requirement from one of the operators.

Reply:
Unfortunately it cannot. Kconfig does not resolve symbols when parsing these.